src/components/MenuButtons.js

```diff
@@ -1,44 +1,117 @@
 import Link from 'next/link';
+import styled from '@emotion/styled';
 import {
   Button,
   List,
   ListItem,
 } from '@material-ui/core';
+import Grid from '../utils/Grid';
```

Comment:
You should also be able to group any import from `utils` together so you could replace this and line 10 with the following:
`import { colors, Grid } from '../utils';`

However, I'm realizing just now that the `colors` module isn't a "component" :/. My bad because I think I had told you to put it in there! You could just put it in the `styles` directory instead.

Reply:
Actually I take it back, I don't think you can do the import I was suggesting based on how the files are structured. But we can still move `colors`